src/scss/index.scss

```diff
@@ -112,3 +112,7 @@ html {
     }
   }
 }
+
+#slider {
```

Comment:
I would rather we investigate why it is overflowing rather than hiding it. Do you have an idea why this is happening? @karatechops @ericsoderberghp I think I can use your help here

Reply:
I'm not totally sure why the range slider's overflow is hidden. Perhaps it was being used for charts somewhere? It's still in [Grommet core](https://github.com/grommet/grommet/blob/7a012d5fc5024650982cbc03722b52e68b515640/src/scss/grommet-core/_base.input.scss#L60). It might be worth opening an issue in [Grommet](https://github.com/grommet/) to investigate, maybe @ericsoderberghp knows more details as he wrote the style. Some context in a comment should be sufficient to push this through, IE `// Grommet hides overflow-x in the range to prevent Foo from Bar.`